kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/DescrNameViewModel.java

```diff
@@ -97,32 +96,36 @@ public Set<ConceptEntity> findAllStatuses(ViewProperties viewProperties) {
     }
 
     public Set<ConceptEntity> findAllCaseSignificants(ViewProperties viewProperties) {
-
-
-        //FIXME after connect-a-thon put this query back
-//        IntIdSet caseSenseDescendents = viewProperties.calculator().descendentsOf(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE.nid());
-//        Set<ConceptEntity> allCaseDescendents = caseSenseDescendents.intStream()
-//                .mapToObj(caseNid -> (ConceptEntity) Entity.getFast(caseNid))
-//                .collect(Collectors.toSet());
-
-        return CASE_SIGNIFICANCE_OPTIONS;
```

Comment:
if we are able to refactor and not use CASE_SIGNIFICANCE_OPTIONS, then we can remove it from DataModelHelper

Reply:
Yes, the new starter data has the case significance options. I have uploaded the starter data in the story.
Also, Rohit said he will follow-up with data team for new database. May-connectathon data is not valid.

Reply:
I was suggesting that we remove this from DataModelHelper... once you removed that return statement on line 108, then CASE_SIGNIFICANCE_OPTIONS is no longer referenced and can be removed.
https://github.com/ikmdev/komet/blob/ef95424ad6cad110509b8958a80bbfea436cd854/kview/src/main/java/dev/ikm/komet/kview/lidr/mvvm/model/DataModelHelper.java#L95

Reply:
if this needs to be reverted based on leaving concept alone then disregard this comment.